flow/layers/clip_path_layer.cc

```diff
@@ -12,8 +12,8 @@
 
 namespace flutter {
 
-ClipPathLayer::ClipPathLayer(Clip clip_behavior)
-    : clip_behavior_(clip_behavior) {
+ClipPathLayer::ClipPathLayer(const SkPath& clip_path, Clip clip_behavior)
+    : clip_path_(clip_path), clip_behavior_(clip_behavior) {
```

Comment:
Does this deeply copy the path? Is that fast enough?

Reply:
I think `SkPath` is copy on write. In any case, this is no slower than our old `setPath`.